wpilibc/src/main/native/include/frc/event/BooleanEvent.h

```diff
@@ -84,7 +84,7 @@ class BooleanEvent {
    * Composes this event with another event, returning a new event that is
    * active when both events are active.
    *
-   * <p>The new event will use this event's polling loop.
+   * <p>The events must use the same event loop.
```

Comment:
I think it's preferable to specify what the consequences rather than stating unenforced assumptions.